src/lib/components/Helpers/history.js

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from 'history';
+
+export const History = createBrowserHistory();
```

Comment:
@KipSigei I think you had the right idea to have this in gisida so it can be the router history no matter what router a user implements, even if they don't use React!